modules/EventEmitter.js

```diff
@@ -520,17 +522,21 @@ class EventEmitter {
                         response: data.response,
                     });
                 } else {
+                    const graphObject = {};
+                    Object.assign(graphObject, { vertices, edges });
+                    const otJson = await this.otJsonImporter.getImport(data_set_id);
```

Comment:
Is there a way that we can get otJson size and data hash after import, before process import method is called